recipe/meta.yaml

```
@@ -36,18 +39,25 @@ requirements:
     - setuptools
   run:
     - python
+    # https://github.com/google-deepmind/tree/blob/{{ version }}/setup.py
+    - absl-py >=0.6.1
+    - attrs >=18.2.0
+    - numpy >=1.21
+    - numpy >=1.21.2  # [py>=310]
+    - numpy >=1.23.3  # [py>=311]
+    - numpy >=1.26.0  # [py>=312]
+    - numpy >=2.1.0  # [py>=313]
```

This is all equivalent to the following, it literally just reflects the first numpy version to support a given python version. The constraint is self-enforcing, because we of course don't have earlier numpy builds on those python versions. So let's keep this simple.
```suggestion
    - numpy >=1.21
```